cloud-operator/values.yaml

```diff
@@ -32,6 +32,35 @@ onboardingSecret:
   clientId: ""
   clientSecret: ""
 
+falco:
+  enabled: true
+  namespace: falco
+  falco:
+    http_output:
+      enabled: true
+      keep_alive: true
+      url: "http://illumio-cloud-operator.illumio-cloud.svc.cluster.local:5000/"
+      insecure: true
+    json_output: true
+    json_include_tags_property: false
+    json_include_output_property: true
+    buffered_outputs: true
+  customRules:
+    rules-networking.yaml: |-
```

Comment:
This may not be the finalized rule or only rule at the end of the day. But this is good for now.

Reply:
LGTM, thanks!